### PR TITLE
Fixes BingMapsImageryProvider ineffective culture option problem.

### LINKED ISSUE #11695: Cesium map labels only displaying in English
## The problem
After setting up Cesium, it works perfecty, but I would like to have the maps in Spanish.
I found that I can do that with the `BingMapsImageryProvider` and its member `culture`, so I came up with this.
```javascript
var imageryProvider = new Cesium.BingMapsImageryProvider({
    url: '//dev.virtualearth.net',
    culture: "es-ES",
    mapStyle: Cesium.BingMapsStyle.AERIAL_WITH_LABELS_ON_DEMAND,
});
console.log(imageryProvider);

$scope.map.scene.imageryLayers.removeAll();
$scope.map.scene.imageryLayers.addImageryProvider(imageryProvider, 0);

$scope.$apply();
```

As that console.log outputs, the culture is being saved correctly:
```javascript
Object { _key: "my_key", _resource: {…}, _mapStyle: "AerialWithLabelsOnDemand", _culture: "es-ES", etc...}
```

But, using one of the images as example, the petition url is still in English: [url](https://t2.ssl.ak.dynamic.tiles.virtualearth.net/comp/ch/02?n=z&mkt=en-US&it=A,G,L&og=2359)
(Manually changing them to `es-ES` works, that proves that the Spanish translation works)

Any help or guidance is apreciated.

### PR DESCRIPTION
**Pull request by ateshuseyin**
# Description
This PR fixes #11695 

According to Cesium documentation (To construct a BingMapsImageryProvider, call [BingMapsImageryProvider.fromUrl](https://cesium.com/learn/cesiumjs/ref-doc/BingMapsImageryProvider.html#.fromUrl). Do not call the constructor directly.) I modified the code as below;
```javascript
const viewer = new Cesium.Viewer("cesiumContainer");

const bing = await Cesium.BingMapsImageryProvider.fromUrl(
  "https://dev.virtualearth.net", {
    culture: "es-ES",
    key: "use-your-own-key",
    mapStyle: Cesium.BingMapsStyle.AERIAL_WITH_LABELS_ON_DEMAND,
});

viewer.scene.imageryLayers.removeAll();
viewer.scene.imageryLayers.addImageryProvider(bing);
```
When I evaluated the problem I saw that the problem was about metadata request and it did not tested well.  I just added culture parameter into metadata request which fixes the problem.

image

## Issue number and link

Fixes #11695 11695

# Author checklist

- [x] I have submitted a Contributor License Agreement
- [x] I have added my name to `CONTRIBUTORS.md`
- [x] I have updated `CHANGES.md` with a short summary of my change
- [x] I have added or updated unit tests to ensure consistent code coverage
- [ ] I have update the inline documentation, and included code examples where relevant
- [x] I have performed a self-review of my code

### CODE REVIEW

File: packages/engine/Source/Scene/BingMapsImageryProvider.js [hunk 1]
```diff
@@ -500,6 +500,10 @@ BingMapsImageryProvider.fromUrl = async function (url, options) {
     queryParameters.mapLayer = options.mapLayer;
   }
 
+  if (defined(options.culture)) {
```
Comment (ggetz) [inline packages/engine/Source/Scene/BingMapsImageryProvider.js:503]: Please document the new `culture` property under the `BingMapsImageryProvider.ConstructorOptions` type definition.

**Review by ggetz — COMMENTED** [review]
Awesome, thanks for the fix @ateshuseyin!

File: packages/engine/Source/Scene/BingMapsImageryProvider.js (hunk 1 above)
Comment (ateshuseyin) [reply packages/engine/Source/Scene/BingMapsImageryProvider.js:503]: Thanks for your quick review. `culture` property under `BingMapsImageryProvider.ConstructorOptions` type is already documented but it was ineffective. I just fixed it.

https://cesium.com/learn/cesiumjs/ref-doc/BingMapsImageryProvider.html#.ConstructorOptions

Comment (ggetz) [reply packages/engine/Source/Scene/BingMapsImageryProvider.js:503]: Fantastic, thanks!

**Comment by ggetz** [comment]
Thanks @ateshuseyin! Congrats on your first merged PR! 🎉

**Pull request merged**